modules/ui/mw.cx.ui.TranslationView.js

```
@@ -13,9 +13,11 @@
  */
 mw.cx.ui.TranslationView = function ( config ) {
 	this.infobar = new mw.cx.ui.Infobar( config );
+	var sourceLanguage = config.sourceLanguage;
```

ESLint

[error] Unexpected var, use let or const instead.

(no-var)